docs/topics/mta-web-reviewing-apps.adoc

```diff
@@ -11,16 +11,22 @@ You can use the {ProductName} ({ProductShortName}) {WebName} to determine the mi
 .Procedure
 
 . In *Migration* view, click *Application inventory*.
-+
-// ![](/Tackle2/AppAssessAnalyze/AnalysisSelect.png)
 . Select the application you want to review.
 +
 [NOTE]
 ====
 Only one application can be reviewed at a time.
 ====
-. Click  *Review*.
-. Select *Proposed action* and *Effort estimate* from the lists. Set *Business criticality* and *Work priority*.
-. Click *Submit review*.
+
+. Click image:kebab.png[] (three dots) at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.
```

Comment:
@sbeskin-redhat - please correct me if I am wrong here but is this not the `Options` menu

Reply:
@anarnold97 Yes, correct. It should be "the Options menu {kebab}".  
{kebab} is in the attributes - :kebab: image:kebab.png[title="Options menu"]